### ISSUE TITLE
Bulk insert fails given event count is not a multiple of 10000

### ISSUE THREAD

**Issue by arkilic** [synchbot/metadataclient#9]
This fails in unit tests with ```TypeError```. Seems like events[] does not get an integer

```
    data_len = len(events)
    chunk_count = data_len // chunk_size + bool(data_len % chunk_size)
    # TODO: Fix bulk insert
    chunks = (events[j * chunk_size:(j+1)*chunk_size] for j in range(int(chunk_count)))
```

**Comment by tacaswell** [comment]
I think that cast to int is the right fix?

**Comment by arkilic** [comment]
I don't think that's the fix. First of, the vals passed are all in. The error persists even after casting like below:
``` 
 chunks = (events[j * int(chunk_size):int(j+1*chunk_size)] for j in range(int(chunk_count)))
```

**Comment by arkilic** [comment]
Also, in this case events are dequeue. Converting to list does not make any difference

**Comment by tacaswell** [comment]
Can you paste in the full traceback?

**Comment by arkilic** [comment]
It was the tip of a large set of problems. @hhslepicka and I fixed it. I will merge his changes to my PRs and problem solved.

**Comment by hhslepicka** [comment]
Fixed by PR #8 .
